### PR TITLE
feat: add permissions docs page

### PR DESCRIPTION
**Pull request by kaushalkapasi**
- adds simple Permissions docs page

### CODE REVIEW

File: docs/extras/permissions/permissions.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Permissions
+sidebar_position: 1
+---
+
+# Permissions
+
+At DevCycle, we offer Permissions gating to protect Environments which are marked as Production from unauthorized changes. This allows Organizations to manage which users can modify the Targeting Status, Rules and Served Variations in a Production Environment that may impact their End Users.
```
Comment (ajwootto) [inline docs/extras/permissions/permissions.md:8]: either `permissions gating` or `Permissions Gating`

Comment (ajwootto) [inline docs/extras/permissions/permissions.md:8]:
```suggestion
At DevCycle, we offer Permissions gating to protect Environments which are marked as "production" from unauthorized changes. This allows Organizations to manage which users can modify the Targeting Status, Rules and Served Variations in a production Environment that may impact their End Users.
```

File: docs/extras/permissions/permissions.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+### Members
+
+- Configure Development and Staging Environments
+- Configure **Inactive** Production Environments
+
+### Publishers
+
+Everything **Members** can do, plus
+
+- Change Targeting Status of Production Environments
```
Comment (ajwootto) [inline docs/extras/permissions/permissions.md:29]: will people really know what this means? Is it simpler to just say "turn features on and off" or something

File: docs/extras/permissions/permissions.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- Modify **Active** Production Environmentss
+
+### Owners
+
+Everything **Publishers** can do, plus
+
+- Change and Manage Roles of all Users in the Organization
```
Comment (ajwootto) [inline docs/extras/permissions/permissions.md:36]:
```suggestion
- Change and manage Roles of all Users in the Organization
```

File: docs/extras/permissions/permissions.md (hunk 2 above)
Comment (kaushalkapasi) [reply docs/extras/permissions/permissions.md:29]: good point, how does this sound? `Enable or Disable Production Environment Targeting`

Comment (ajwootto) [reply docs/extras/permissions/permissions.md:29]: better, though I also worry that "disable production targeting" could be interpreted to mean "there are no rules now its just on or off for everyone"

Comment (cdruxerman) [reply docs/extras/permissions/permissions.md]: I think that's an unlikely interpretation, so i'm not too worried about it

**Review by cdruxerman — COMMENTED** [review]
Added some edits in comment

File: docs/extras/permissions/permissions.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+:::
+
+## What Can Users Do?
+
+For more information, checkout the [Organization Roles](/essentials/organizations#organization-roles) documentation.
+
+### Members
+
+- Configure Development and Staging Environments
+- Configure **Inactive** Production Environments
```
[the remark this answers is not in the document]
Comment (kaushalkapasi) [reply docs/extras/permissions/permissions.md:23]: I'll add it in here 🙏 as a limitation for Members

File: docs/extras/permissions/permissions.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+title: Permissions
+sidebar_position: 1
+---
+
+# Permissions
+
+At DevCycle, we offer Permissions Gating to protect Environments which are marked as "production" from unauthorized changes. This allows Organizations to manage which users can modify the Targeting Status, Rules and served Variations in a production Environment that may impact their end users.
```
[the remark this answers is not in the document]
Comment (ajwootto) [reply docs/extras/permissions/permissions.md:8]: what additional context do you want to provide with that first paragraph? it seems like it's mostly just drawing attention to the limited capabilities here

File: docs/extras/permissions/permissions.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+To ensure usability, all users are able to make any edits to a Feature when production targeting isn't enabled. Permissions Gating and it's protections come into play when production targeting is either enabled or a user is attempting to enable production.
+
+See below for the full details on what each user role can do when permissions are enabled.
+
+### Members
+
+- ✅ Configure Development and Staging Environments
+- ✅ Configure **Inactive** Production Environments
+- ❌ Configure **Active** Production Environments, Variables and Variations
```
Comment (cdruxerman) [inline docs/extras/permissions/permissions.md:28]: @kaushalkapasi can you split variables and variations out into their own line and write
"When a production environment's targeting is enabled, all editing of Variables and Variations is disabled for members

Comment (kaushalkapasi) [reply docs/extras/permissions/permissions.md:28]: Going to use:
`- ❌ When Production is **Active**, Configure Variables and Variations`

**Review by cdruxerman — APPROVED** [review]

**Pull request merged**